levels/Level0/Module4/extras.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<title>Level 0 - Module 4 Extras</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://league-central.github.io/curriculum/style/style.css">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway">
+<style>
+body,h1 {font-family: "Raleway", sans-serif}
+body, html {height: 100%}
+.bgimg {
+    min-height: 100%;
+    background-position: center;
+    background-size: cover;
+}
+</style>
+    <script src="https://league-central.github.io/curriculum/script/headerAndFooter.js"> </script>
+
+    <body>
+        <div id="wrap">
+            <div id="main">
+                <div id = "linkPageBody">
+                
+    <h1>Level 0 Module 4</h1>
+   
+       <hr class="w3-border-grey" style="margin:auto;width:40%">
+        
+       <div id="leftDiv">
+       <h2>Extra Practice</h2>    
+       <h3><a href="../Recipes/PinTheTail.html">Pin the Tail</a> (Processing)</h3>
+        <h3><a href="../Recipes//HideTheCreeper.html">Hide the Creeper</a> (Processing)</h3>
```

Comment:
Link doesn't work unless the extra '/' is removed:
"../Recipes/HideTheCreeper.html"